Checkout/Checkout.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name         = "Checkout"
+  s.summary      = "Checkout Core Features that can only be used with the right level of PCI Compliance. Please contact us for the details."
```

Comment:
@patrick-hoban-cko Can you review this text? if already done please ignore.

Reply:
Checkout Core functionality can only be used with the right level of PCI Compliance. Please contact us for more details.